pipelines/broad/rna_seq/RNAWithUMIsPipeline.changelog.md

```diff
@@ -1,3 +1,12 @@
+# 1.0.2
+2022-02-xx  (Date of Last Commit)
+
+* Updated STAR command line argument
+(TODO) Updated ribosomal intervals
```

Comment:
Can you clarify a bit more here why you're updating the command line argument or what you're updating it to do now that it didn't do before? And same with the updates to the intervals, reflat etc like if it's that you're moving to a new annotation version just say that and say what the version is. Thanks!

Reply:
Sounds good, will do